src/ralph/backends/data/async_es.py

```diff
@@ -114,6 +114,7 @@ async def read(  # noqa: PLR0913
         chunk_size: Optional[int] = None,
         raw_output: bool = False,
         ignore_errors: bool = False,
+        max_statements: Optional[int] = None,
```

Comment:
Perhaps we could use `PositiveInt` ?

Reply:
Good point) It is indeed more explicit to use `PositiveInt`) Updated it)